### PR TITLE
Feature/menu in drawer

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Account menu in a drawer

### CODE REVIEW

**Comment by huayunh** [comment]
Functionality-wise looks nice, but I wonder why the icon stays black in an actively selected drawer item.
image

**Comment by emclaug2** [comment]
@huayunh This is a actually a bug in the themes repo, introduced in version v6.1.1 of @pxblue/angular-themes

I've made a new issue for this.
https://github.com/pxblue/angular-themes/issues/22

**Review by jeffvg — APPROVED** [review]
looks good to me

File: src/app/pages/account-menu/in-a-drawer/menu-in-a-drawer.component.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                            (select)="select(navItem)"
+                        >
+                            <mat-icon pxb-icon>{{ navItem.icon }}</mat-icon>
+                        </pxb-drawer-nav-item>
+                    </pxb-drawer-nav-group>
+                </pxb-drawer-body>
+            </pxb-drawer>
+        </mat-sidenav>
+        <mat-sidenav-content class="content">
+            <mat-toolbar data-cy="pxb-toolbar" color="primary" class="app-bar mat-elevation-z2">
```
Comment (huayunh) [inline src/app/pages/account-menu/in-a-drawer/menu-in-a-drawer.component.html:43]: z4

**Review by huayunh — APPROVED** [review]

File: src/app/pages/account-menu/in-a-drawer/menu-in-a-drawer.component.html (hunk 1 above)
Comment (emclaug2) [reply src/app/pages/account-menu/in-a-drawer/menu-in-a-drawer.component.html]: You got it!

**Pull request merged**